frontend/src/Store/Actions/authorEditorActions.js

```diff
@@ -30,6 +31,58 @@ export const defaultState = {
   filters,
   filterPredicates,
 
+  columns: [
+    {
+      name: 'status',
+      columnLabel: 'Status',
+      isSortable: true,
+      isVisible: true,
+      isModifiable: false
+    },
+    {
+      name: 'sortName',
+      label: 'Name',
+      isSortable: true,
+      isVisible: true
+    },
+    {
+      name: 'qualityProfileId',
+      label: 'Quality Profile',
+      isSortable: true,
+      isVisible: true
+    },
+    {
+      name: 'metadataProfileId',
+      label: 'Metadata Profile',
+      isSortable: true,
+      isVisible: false
+    },
+    {
+      name: 'albumFolder',
+      label: 'Album Folder',
```

Comment:
Remove, AlbumFolder was Lidarr

Reply:
@Qstick this should become `bookFolder` right?

Reply:
Maybe, I don't recall if we kept that option in Readarr. I'd have to look at the Author model

Reply:
That’s alright I will have a look later on

Reply:
it wasn't in the model so I have removed it